unstructured/file_utils/ndjson.py

```diff
@@ -0,0 +1,24 @@
+import json
+
+
+def dumps(obj: list[dict], **kwargs) -> str:
+    lines = []
+    for each in obj:
+        lines.append(json.dumps(each, **kwargs))
+    return "\n".join(lines)
```

Comment:
A matter of style, but this can be:
```python
return "\n".join(json.dumps(each, **kwargs) for each in obj)

Which is both more concise and more efficient.